apps/pharos_web/web/core/real_search.ex

```diff
@@ -3,19 +3,40 @@ defmodule Core.RealSearch do
   def execute(%{"topic" => topic, "amount" => amount}) do
     query = %{topic: topic, amount: String.to_integer(amount)}
 
```

Comment:
Wasn't sure how much info we wanted to put in this file. For the starter code we don't want them to use distributed nodes, so I popped in the code to call it manually and commented the rest as 'stage 2'. 

Also do we need to write unit tests for this, and what about when it changes to stage 2 code?